test/connectors/CustomsDeclareExportsConnectorSpec.scala

```diff
@@ -39,12 +37,9 @@ import unit.base.UnitSpec
 
 import scala.concurrent.ExecutionContext.global
 
-class CustomsDeclareExportsConnectorSpec extends UnitSpec with ScalaFutures {
+class CustomsDeclareExportsConnectorSpec extends UnitSpec with ScalaFutures with WithInjector {
```

Comment:
`with WithInjector` seems a little strange, why not just `with Injector`?

Or if that has name clashes maybe `with InjectorSugar` or something?